### PR TITLE
Project bootstrap

### PR DESCRIPTION
**Pull request by LeoCBS**
Just creating project README and init files

@mateuspmello

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/LeoCBS/httpmiddleware
+
+go 1.17
```
Comment (danielfmelo) [inline go.mod:3]: You are testing with go 1.20 but the module requires 1.17. Could it have some inconsistency in the future?

File: middleware_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	"github.com/LeoCBS/httpmiddleware"
+	"github.com/julienschmidt/httprouter"
+	"github.com/sirupsen/logrus"
+)
+
+func TestNewMiddleware(t *testing.T) {
+	l := logrus.New()
+	router := httprouter.New()
+	_ = httpmiddleware.New(l, router)
```
Comment (danielfmelo) [inline middleware_test.go:17]: Maybe assert the return as not nil?

File: go.mod (hunk 1 above)
Comment (LeoCBS) [reply go.mod:3]: good point, i will change this value on next PR :) @danielfmelo

File: middleware_test.go (hunk 2 above)
Comment (LeoCBS) [reply middleware_test.go:17]: good point, i will add this assert on next PR :) @danielfmelo

**Pull request merged**